src/celeritas/ext/GeantImporter.cc

```diff
@@ -266,14 +266,38 @@ store_materials(GeantImporter::DataSelection::Flags particle_flags)
     std::vector<ImportMaterial> materials;
     materials.resize(g4production_cuts_table.GetTableSize());
 
-    std::vector<G4ProductionCutsIndex> particle_cuts;
+    using CutRange = std::pair<G4ProductionCutsIndex,
+                               std::unique_ptr<G4VRangeToEnergyConverter> >;
+
+    std::vector<CutRange> cut_converters;
```

Comment:
Is there an advantage in creating a `vector<pair<...>>` instead of `map<...>`?

Reply:
Not meaningfully: I was still thinking of this in terms of "iterate over the ordered g4 indices" as a vector with elements removed, but a map would work just as fine (though it takes up slightly more memory and will be slightly slower, it's not going to be meaningfully different for this case).

Reply:
Thinking about this: the way it's used (or intended to be used) matters. We only ever iterate over this container, so it could be as simple as a list. If we needed fast indexed access, it needs to be a vector. If we want fast key-value access, it should be a map. Since `std::list` doesn't have any advantages over small vectors, it's better to leave this as a vector.